src/components/SimplePriceChart/SimplePriceChart.tsx

```diff
@@ -37,7 +40,42 @@ const MarketDataChart: React.FC<SimplePriceChartProps> = ({
     const {
       payload: { x, y },
     } = chartData
-    return [new Date(x).toLocaleDateString(), '$' + formatFloats(y)]
+    let timeString = new Date(x).toLocaleDateString()
+    if (durationSelector === 0) {
+      timeString = new Date(x).toLocaleTimeString([], {
+        hour: 'numeric',
+        minute: 'numeric',
+      })
+    }
+    return [timeString, '$' + formatFloats(y)]
+  }
```

Comment:
Maybe look into optimizing the above section? Could try memoizing `{ x: localeDateString }`